src/mudlet.cpp

```diff
@@ -1431,6 +1431,29 @@ void mudlet::closeHost(const QString& name)
     int hostCount = mHostManager.getHostCount();
     emit signal_hostDestroyed(pH, --hostCount);
     mHostManager.deleteHost(pH->getName());
+    updateMultiviewControlsEnabled();
+}
+
+void mudlet::updateMultiviewControlsEnabled()
```

Comment:
```suggestion
void mudlet::updateMultiviewControls()
```

Reply:
:-1: To me the shorter wording implies that this method would modify the checked/unchecked state - which does need to happen on occasions - but is *not* done by this method.

Reply:
I don't think anyone will get massively confused by it and will make grievous errors. People will appreciate if they don't have to work with very long names, though.

Reply:
How does the 'enabled' clear this up if it's left in place, though?

Reply:
It is updating the "Enabled" states/flags of the multi-view controls - not (as might be assumed from the shorter name) the "Checked/Activated" ones....

Reply:
I agree that not adding the enabled creates a wrong impression. However, the enabled confuses me here as well (which might be from my missing experience...). To me the name doesn't actually seem that long.

Reply:
It was a *tad* shorter than `updateMultiviewControlsEnabledStates()`... :man_shrugging:

Reply:
:angry: So I listened to this discussion, made some changes - but you ignored me and changed this one afterwards to your own *overly short* (and I believe **misleading**) suggestion @vadi2 ...!

Reply:
I looked at `updateEnableOfMViewControls`, had no idea what it meant, thought of a shorter one and ended up arriving at the original without knowing it... sorry for overruling you, but we've got to keep stuff understandable for everyone involved. I don't believe I'm the only one who would say that `updateEnableOfMViewControls` is not a super intuitive name.